lib/capistrano/data_migrate/migrate.rb

```diff
@@ -0,0 +1,42 @@
+Rake::Task["deploy:migrate"].clear
```

Comment:
I think that the `clear` call is unnecessary. I'm doing some tests and will update the docs.